.github/workflows/npm-cd.yml

```diff
@@ -139,15 +134,40 @@ jobs:
                   npm_scope: ${{ vars.NPM_SCOPE }}
                   publish: "true"
                   github-token: ${{ secrets.GITHUB_TOKEN }}
+            
+            - name: Del package.json.tmpl
+              shell: bash
+              working-directory: ./node
+              run: |
+                rm package.json.tmpl
+                echo $(ls *json*) 
+
+            - name: Check if RC and set a distribution tag for the package
+              shell: bash
+              run: |
+                  if [[ "${GITHUB_REF:11}" == *"rc"* ]]
+                  then
+                    echo "This is a release candidate"
+                    export npm_tag="next"
+                  else
+                    echo "This is a stable release"
+                    export npm_tag="latest"
+                  fi
+                  echo "NPM_TAG=${npm_tag}" >> $GITHUB_ENV
+
+            - name: Check that the release version dont have typo init
+              if: ${{ github.event_name != 'pull_request' && contains(github.ref, '-') && !contains(github.ref, 'rc') }}
+              run: exit 1
 
             - name: Publish to NPM
               if: github.event_name != 'pull_request'
               shell: bash
               working-directory: ./node
               run: |
+                  npm pkg fix
                   set +e
                   # Redirect only stderr
-                  { npm_publish_err=$(npm publish --access public 2>&1 >&3 3>&-); } 3>&1
+                  { npm_publish_err=$(npm publish --tag ${{ env.NPM_TAG }} --access public 2>&1 >&3 3>&- | grep -v "npm notice" | grep -v "warn" | grep -v "WARN"); } 3>&1
```

Comment:
It discards it from being shown as an error and fails the step. It doesn't discard the log.
For some reasons I don't understand some logs going to stderr instead of stdout even though they are not errors. 
I dont want to fail on npm notice log, which is an harmful log that for some reason go to stderr and fail the cd, neither on warn like "i just fixed your dependency hell" or "some packages you use not the latest version", we want to fail on error.
We can later on after GA get deeper and understand why the logs directed to stderr falsely, but i don't think it's currently critical.
I can open an issue for a post GA task.

Reply:
3>&1 is swapping between STDERR and STDOUT, so this is probably why non-error output returns as an error. 
according to:
https://unix.stackexchange.com/questions/474177/how-to-redirect-stderr-in-a-variable-but-keep-stdout-in-the-console
`{ err=$(cmd 2>&1 >&3 3>&-); } 3>&1` as we do should solve this issue, but maybe your added code (maybe the curly bracelets) are breaking this logic. So we need to fix it / use another method to capture STDERR without swapping STDOUT